frontend/src/common/hooks/useCategoryFilter/common/multiPanelOntologyUtils.ts

```diff
@@ -341,8 +346,19 @@ function buildOntologyPanelCategoryViews(
         )
       : panelUINodes;
 
-    // Collect the select category values for the values to be included.
-    const selectCategoryValues = includeUINodes
+    // Create a set of visibleNodeIds to use in the next step where
+    // we mark each node as visible or not.
+    const visibleSet = visibleUINodeIds.reduce(function (
```

Comment:
nit: prefer () => {} for anonymous function 🙏 Thank you!

Reply:
discussion: for constructing the `visibleSet`, what do you think about the following 😄 ?

```ts
const visibleSet = new Set(visibleUINodeIds.map((node) => node.categoryValueId))
```

I'm hoping this would improve readability 🤞

Reply:
Will fix.

Reply:
Updated thx.